src/NodaTime.Web/wwwroot/css/site.css

```diff
@@ -260,29 +265,27 @@ footer .design {
   font-style: italic;
 }
 
-.accordion footer {
-  background-color: #fff;
-  padding: 10px 16px;
-  font-size: 12px;
-  border: 0px none;
+.side-nav {
+  padding-bottom: 0;
 }
 
-.accordion-navigation {
+.side-nav > .heading {
   background-color: #FFE76B;
```

Comment:
It's good practice to specify `color` and `background-color` together unless you really want to inherit one. This is picking up a default (#008cba) from Foundation, from what I can see.

Also, that combination isn't super readable to my eyes (and it's got a contrast ratio < the recommended for small text). Perhaps specify #006b8f here instead? It's the same hue, but higher contrast.

Reply:
Happy to.